app/livechat/server/lib/stream/screenSharingStream.js

```diff
@@ -0,0 +1,6 @@
+import { Meteor } from 'meteor/meteor';
+
+export const screenSharingStreamer = new Meteor.Streamer('screenhsaring');
+screenSharingStreamer.allowRead('all');
```

Comment:
```suggestion
screenSharingStreamer.allowRead('all');
```
`all`? I don't think so. you need to check the `hasPermission(this.userId, 'view-l-room')`.